basic-steps/src/main/resources/org/jenkinsci/plugins/workflow/steps/MailStep/config.jelly

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+The MIT License
+
+Copyright 2014 Jesse Glick.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+    <f:entry field="from" title="From">
+        <f:textbox />
+    </f:entry>
+    <f:entry field="to" title="To">
+        <f:textbox />
+    </f:entry>
+    <f:entry field="cc" title="CC">
+        <f:textbox />
+    </f:entry>
+    <f:entry field="bcc" title="BCC">
+        <f:textbox />
+    </f:entry>
+    <f:entry field="subject" title="Subject">
+        <f:textbox />
+    </f:entry>
+    <f:entry field="body" title="Body">
+        <f:textbox />
+    </f:entry>
+    <f:entry field="replyTo" title="Reply-To">
```

Comment:
Some of these could probably be inside `f:advanced` since there is not commonly a need to customize them.

Should also create `help-replyTo.html` etc. where it is not obvious what the field is for, especially when it is not obvious what happens when it is left blank.